lib/versioned/runner.js

```diff
@@ -71,7 +71,14 @@ function installPackages(cb) {
   tryInstall(0)
 
   function tryInstall(attempts) {
-    var args = ['install', '--no-save', '--no-package-lock']
+    var args = [
+      'install',
+      '--no-save', // do not update package file
+      '--no-package-lock', // do not update package-lock file
+      '--no-audit', // skip audit output
+      '--no-fund', // skip funding output
+      '--legacy-peer-deps' // skip installing peer deps
```

Comment:
i need to look into the test run from the agent because I think we do need this option

Reply:
Looking into this test it might be wrong?

the deps are
```
    "dependencies": {
        "ejs": "2.5.5",
        "hapi": ">=17.0.0",
        "vision": "5.1"
      },
```

but the folder implies this should only be hapi 17 but it is also running against hapi 18.  This fails with npm 7 because vision 5.1 has a peer dep of  https://github.com/hapijs/vision/blob/v5.1.0/package.json#L22. should I open a separate PR to lock down hapi to `>=17.0.0 < 18.0.0`